lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -59,22 +59,16 @@
     version_added: "2.2"
   wait:
     description:
-      - Wait for the copied AMI to be in state 'available' before returning.
+      - wait for the copied AMI to be in state 'available' before returning.
     required: false
-    default: "no"
-    choices: [ "yes", "no" ]
-  wait_timeout:
-    description:
-      - How long before wait gives up, in seconds.
-    required: false
-    default: 1200
```

Comment:
You cannot just remove wait_timeout, if we do need to remove this it should be deprecated first to allow users to migrate